Test/E2ETests/E2ETests/Test452Base.cs

```diff
@@ -74,12 +74,13 @@ public static void MyClassInitializeBase()
             GetDockerCompose();
             Trace.WriteLine("Getting docker-compose.exe completed.");
 
-            //DockerUtils.RemoveDockerImage(Apps[AppNameBeingTested].imageName, true);
-            //DockerUtils.RemoveDockerContainer(Apps[AppNameBeingTested].containerName, true);
+            DockerUtils.RemoveDockerImage(Apps[AppNameBeingTested].imageName, true);
+            DockerUtils.RemoveDockerContainer(Apps[AppNameBeingTested].containerName, true);
 
             // Deploy the docker cluster using Docker-Compose
-            //DockerUtils.ExecuteDockerComposeCommand("up -d --force-recreate --build", DockerComposeFileName);
-            DockerUtils.ExecuteDockerComposeCommand("up -d --build", DockerComposeFileName);
+            DockerUtils.ExecuteDockerComposeCommand("up -d --force-recreate --build", DockerComposeFileName);
```

Comment:
if we are force-recreating, the the above steps to remove image and container is not required.
but if this is helping anyway - lets just do it.